sdk/servicebus/azure-servicebus/migration_guide.md

```diff
@@ -1,164 +1,444 @@
+message.enqueued_time_utc
+message.scheduled_enqueue_time_utc
+```
 
 ### Working with AutoLockRenewer
-| In v0.50 | Equivalent in v7 | Sample |
-|---|---|---|
-| `azure.servicebus.AutoLockRenew().register(message)` | `azure.servicebus.AutoLockRenewer().register(receiver, message)` | [Auto lock renew a message received from a non-sessonful entity](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples/sync_samples/auto_lock_renew.py) |
-| `azure.servicebus.AutoLockRenew().register(session_receiver)` | `azure.servicebus.AutoLockRenewer().register(session_receiver, session_receiver.session)` | [Auto lock renew a session for sessionful receiver](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/servicebus/azure-servicebus/samples/sync_samples/auto_lock_renew.py) |
-| `azure.servicebus.AutoLockRenew().shutdown()` | `azure.servicebus.AutoLockRenewer().close()` | [Close an auto-lock-renewer](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/servicebus/azure-servicebus/samples/sync_samples/auto_lock_renew.py) |
+
+- `AutoLockRenew` class is renamed to `AutoLockRenewer`.
+- `register` method now takes both the receiver and the object to be auto-lock-renewed.
+- `shutdown` method is renamed to `close` to keep consistent with sender and receiver classes.
+
+In V0.50:
+```python
+from azure.servicebus import AutoLockRenew
+auto_lock_renew = AutoLockRenew()
+
+with receiver:
+    auto_lock_renew.register(received_message)
+
+with session_receiver:
+    auto_lock_renew.register(session_receiver)
+
+auto_lock_renew.shutdown()
+```
+
+In V7:
+```python
+from azure.servicebus import AutoLockRenewer
+auto_lock_renewer = AutoLockRenewer()
+
+with receiver:
+    auto_lock_renew.register(receiver, received_message)
+
+with session_receiver:
+    auto_lock_renew.register(session_receiver, session_receiver.session)
+
+auto_lock_renew.close()
+```
 
 ### Working with Message properties
-| In v0.50 | Equivalent in v7 | Sample |
+
+There're properties on `ServiceBusMessage` being added/removed/changed, please check the table follow:
+
+| In v0.50 | Equivalent in v7 | Note |
 |---|---|---|
 | `azure.servicebus.Message.user_properties` | `azure.servicebus.ServiceBusMessage.application_properties` | Some message properties have been renamed, e.g. accessing the application specific properties of a message. |
 
-## Working with Administration Client
+### Working with Administration Client
 
 The management related operations like creating, updating, deleting and listing queues, topics, subscriptions or rules are to be done with the `ServiceBusAdministrationClient` which can be imported from the `azure.servicebus.management`. If one was relying on `control_client` available in v0.50, and cannot migrate to the `azure-mgmt-servicebus` package (or requires SAS based authentication for management), one would utilize this migration segment, otherwise, the aforementioned dedicated mgmt package should be used.
```

Comment:
We don't have any reasons why we should be telling customers who are using this package to do the management operations to move to the mgmt package. So, we don't need to call out `azure-mgmt-servicebus` here.

```suggestion
In v0.50, you could create/get/update/delete/list Service Bus queues/topics/subscriptions/rules using the `control_client`. In v7, this is replaced by the `ServiceBusAdministrationClient`.
```